classes/Search.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Not needed

Reply:
That was already in the code. I didnt add anything other than the 1 line on 331. My formatter may have moved things around but I made only the 1 line change. any other issues in the code pre-date my involvement.